kzz7yh-a09618/cod-ve07v1_kzz7yh-a09618.xml

```diff
@@ -57,19 +57,19 @@
             <p xml:id="kzz7yh-a09618-d1e103">
       ¶ Articulus. ii. 
       <lb ed="#V" n="102"/>Consequenter querendum est 
-      <lb ed="#V" n="103"/>mou 
-      cogno<lb ed="#V" n="104" break="no"/>scendi deum per creaturam. Circa quod querun 
-      <lb ed="#V" n="105"/>tur tria. Primo vtrum per creaturas 
+      <lb ed="#V" n="103"/>de modo  
+      cogno<lb ed="#V" n="104" break="no"/>scendi deum per creaturam. Circa quod 
+      querun<lb ed="#V" n="105" break="no"/>tur tria. Primo vtrum per creaturas 
       possi<lb ed="#V" n="106" break="no"/>mus aliquo modo conferre de deo quid est. 
       Se<lb ed="#V" n="107" break="no"/>cundo vtrum per creaturas possimus cognoscere quod deus trinus 
       <lb ed="#V" n="108"/>et vnus est. 3osi intellectus beatus deum per creaturam cognoscem potest. 
      </p>
             <div xml:id="kzz7yh-a09618-Dd1e121">
                <head xml:id="kzz7yh-a09618-Hd1e123">Quaestio 1</head>
```

Comment:
Are we mssing a question title here?

Reply:
I don't see a place in the text that specifically says "quaestio x..." but the wording is similar to a traditional question opening.